rfcs/005-json-comments-schemas.md

```diff
@@ -0,0 +1,13 @@
+# Summary
```

Comment:
We can delete this file -- it was helpful while writing the PR but I think it's no longer necessary.

Reply:
resolved in commit 502ad59f3